src/core/evaluator.js

```diff
@@ -85,6 +86,7 @@ const DefaultPartialEvaluatorOptions = Object.freeze({
   useSystemFonts: true,
   cMapUrl: null,
   standardFontDataUrl: null,
+  canvasMaxAreaInBytes: -1,
```

Comment:
For consistency with the ordering in the API, can you please move this line to *just after* `isOffscreenCanvasSupported: false,` instead?